app/js/lib/foxbox.js

```diff
@@ -10,6 +10,8 @@ import FoxboxDb from './foxbox-db';
 const settings = new FoxboxSettings();
 const db = new FoxboxDb();
 
+const QUERY_STRING_AUTH_TOKEN_NAME = 'auth';
```

Comment:
Constants should be moved to `foxbox-settings.js`. See how `REGISTRATION_SERVICE` is implemented for an example.